flow-api/build.gradle

```diff
@@ -33,6 +33,7 @@ dependencies {
   }
   implementation group: 'org.glassfish.jersey.connectors',  name: 'jersey-apache-connector',     version: jerseyCoreVersion
   implementation group: 'net.jcip',                      name: 'jcip-annotations',               version: '1.0'
+  implementation group: 'org.hibernate',                 name: 'hibernate-validator',            version: hibernateValidatorVersion
```

Comment:
why is a hibernate dependency  required in the api module?

Reply:
i will remove this. i thought I was going to use one of hibernate validators to validate the input size, but found better constraints with javax